src/libmongoc/tests/unified/entity-map.c

```diff
@@ -31,6 +31,11 @@
 #define REDUCED_HEARTBEAT_FREQUENCY_MS 500
 #define REDUCED_MIN_HEARTBEAT_FREQUENCY_MS 50
 
+struct _entity_findcursor_t {
```

Comment:
Cursors are lazy in libmongoc, so they do not send the initial command (in this case `find`) until the first iteration.

This updates the implementation of the `createFindCursor` test operation to adhere to the specification:

https://github.com/mongodb/specifications/blob/86197086c2290a67500321f008b7a0a0685f0da7/source/unified-test-format/unified-test-format.rst#createfindcursor

> Test runners for drivers that lazily execute the find command on the first iteration of the cursor MUST iterate the resulting cursor once.